lms/static/sass/discussion/_discussion.scss

```diff
@@ -653,9 +591,9 @@ body.discussion {
       width: 100%;
       height: 40px;
       padding: 0 ($baseline/2);
-      @include box-sizing(border-box);
+      box-sizing: border-box;
```

Comment:
I think the `@include` uses Bourbon and automatically spits out different rules for various browsers.